### DISCUSSION TITLE
[2회] 과제(~02/19)

### DISCUSSION THREAD

**Discussion by junseo511** [like-comment-subscribe-and-Algorithm/sehunTV#4] [General]
1. 알고리즘 3문제 풀기
   - 분할 정복 : 1780
   - 동적 계획법 : 1904
   - 이분 탐색 : 2512

2. 개인문제 자유 7문제 풀기

**Comment by haeti-dev** [comment]
DP 개인문제로 추천합니다. 허세아니고 진짜 쉬움
https://www.acmicpc.net/problem/9095

**Reply by librarywon** [reply]
아 이걸 왜 지금 봤지

**Reply by junseo511** [reply]
저는 2주차에 재밌게 풀었습니당 😁
